index.js

```diff
@@ -37,8 +37,10 @@ function defaults (params) {
     decodeEmails: false,
     // Support gzip encoded responses
     gzip: true,
-    // Adds secure TLSv1.3 ciphers when using older openssl versions
-    agentOptions
+    agentOptions: {
```

Comment:
Could you please also update the docs?

Reply:
I would but do we want to document this? It's a rather advanced option that average users don't need to be aware of or play with when using the library as intended. The brave can always read the source. If you think it needs to be mentioned in the README, I'll document it.

Reply:
Then again, maybe I'm making too many assumptions about the user base. Just let me know and I'll make the changes. :)

Reply:
I would appreciate if you could do that :) otherwise I can take care of it.
I would prefer these settings to be documented well, as they are already
present in the README, so why not to keep them up to date?

Reply:
Updated.